Index: app/src/main/res/drawable/custom_input.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"utf-8\"?>\r\n<selector xmlns:android=\"http://schemas.android.com/apk/res/android\">\r\n    <item android:state_enabled=\"true\" android:state_focused=\"true\">\r\n        <shape>\r\n            <corners android:radius=\"10dp\"/>\r\n            <stroke android:width=\"1dp\" android:color=\"\"\r\n        </shape>\r\n    </item>\r\n</selector>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/drawable/custom_input.xml b/app/src/main/res/drawable/custom_input.xml
--- a/app/src/main/res/drawable/custom_input.xml	(revision 544adb7bea5e647ac222c98bd9bf008336be6eee)
+++ b/app/src/main/res/drawable/custom_input.xml	(date 1681110124736)
@@ -3,7 +3,7 @@
     <item android:state_enabled="true" android:state_focused="true">
         <shape>
             <corners android:radius="10dp"/>
-            <stroke android:width="1dp" android:color=""
+            <stroke android:width="1dp" android:color="wh"
         </shape>
     </item>
 </selector>
\ No newline at end of file
